pkg/systemd/overview-cards/configurationCard.jsx

```diff
@@ -99,11 +99,29 @@ export class ConfigurationCard extends React.Component {
         $("#system_information_ssh_keys").on("hide.bs.modal", () => this.host_keys_hide());
 
         this.realmd.addEventListener("changed", () => this.setState({}));
+
+        $(superuser).on("changed", () => {
+            if (superuser.allowed !== null)
```

Comment:
OOI, can that ever happen? I thought `null` just means "not yet initialized", but on a changed signal it should always be initialized?

Reply:
Yes, it should, good point.